doc/quickstart/test_network.rst

```diff
@@ -6,68 +6,68 @@ Starting a Test Network
 
-The quickest way to start a CCF test network is to use the `start_test_network.sh <https://github.com/microsoft/CCF/blob/master/start_test_network.sh>`_ test script, specifying the :doc:`enclave image </developers/index>` to run.
+The quickest way to start a CCF test network is to use the `sandbox.sh <https://github.com/microsoft/CCF/blob/master/tests/sandbox/sandbox.sh>`_ test script, specifying the :doc:`enclave image </developers/index>` to run.
 
-The script creates a new test CCF network composed of 3 nodes running locally. All the governance requests required to open the network to users are automatically issued.
+The script creates a new one node test CCF network running locally. All the governance requests required to open the network to users are automatically issued.
 
 For example, deploying the ``liblogging`` example application:
 
 .. code-block:: bash
 
     $ cd CCF/build
-    $ ../start_test_network.sh --package ./liblogging.enclave.so.signed
+    $ ../sandbox.sh --package ./liblogging.virtual.so
     Setting up Python environment...
     Python environment successfully setup
-    [14:47:41.562] Starting 3 CCF nodes...
-    [14:48:12.138] Started CCF network with the following nodes:
-    [14:48:12.138]   Node [ 0] = 127.177.10.108:37765
-    [14:48:12.138]   Node [ 1] = 127.169.74.37:58343
-    [14:48:12.138]   Node [ 2] = 127.131.108.179:50532
-    [14:48:12.138] You can now issue business transactions to the ./liblogging.enclave.so.signed application.
-    [14:48:12.138] Keys and certificates have been copied to the common folder: /path/to/test_network_common
-    [14:48:12.138] See https://microsoft.github.io/CCF/master/users/issue_commands.html for more information.
-    [14:48:12.138] Press Ctrl+C to shutdown the network.
+    [16:14:05.294] Starting 1 CCF nodes...
+    [16:14:05.295] Virtual mode enabled
+    [16:14:10.010] Started CCF network with the following nodes:
+    [16:14:10.011]   Node [0] = https://127.0.0.1:8000
+    [16:14:10.011] You can now issue business transactions to the ./liblogging.virtual.so application.
+    [16:14:10.011] Keys and certificates have been copied to the common folder: /data/src/CCF/build/workspace/sandbox_common
+    [16:14:10.011] See https://microsoft.github.io/CCF/master/users/issue_commands.html for more information.
+    [16:14:10.011] Press Ctrl+C to shutdown the network.
 
 .. note::
 
-    - To use CCF `virtual` mode, the same command can be run with ``TEST_ENCLAVE=virtual`` set as environment variable and the virtual version of the enclave application passed to the script. For example ``$ TEST_ENCLAVE=virtual ../start_test_network.sh --package ./liblogging.virtual.so``.
+    - `sandbox.sh` defaults to using CCF's `virtual` mode, which does not require or make use of SGX. To load debug or release enclaves and make use of SGX, `-e` must be set to the right value, for example: `./sandbox.sh -e release -p ./liblogging.enclave.so.signed`
```

Comment:
```suggestion
    - `sandbox.sh` defaults to using CCF's `virtual` mode, which does not require or make use of SGX. To load debug or release enclaves and make use of SGX, `--enclave-type` must be set to the right value, for example: `./sandbox.sh --enclave-type release -p ./liblogging.enclave.so.signed`
```

I prefer to use the long arg names where we have space. Also makes it easier to find this option (and its possible values) in the `--help` output.